rqd/Dockerfile

```diff
@@ -38,7 +38,7 @@ RUN python3.6 -m grpc_tools.protoc \
 
 # Fix imports to work in both Python 2 and 3. See
 # <https://github.com/protocolbuffers/protobuf/issues/1491> for more info.
-RUN sed -i 's/^\(import.*_pb2\)/from . \1/' rqd/rqd/compiled_proto/*.py
+2to3 -wn -f import rqd/rqd/compiled_proto/*_pb2*.py
```

Comment:
Just to check, did you test these changes by doing a local build of the Docker container?

For example for RQD: https://www.opencue.io/docs/getting-started/deploying-rqd/#option-2-building-and-running-rqd-from-source

I believe we may need to add `2to3` to [our top-level `requirements.txt`](https://github.com/AcademySoftwareFoundation/OpenCue/blob/master/requirements.txt) to ensure that library is available within our various environments.

Reply:
no i did not, i'll build it and check and get back to you